### PR TITLE
Update server.js

### PR DESCRIPTION
**Pull request by davidcao1337**
Added connection to MongoDB

### CODE REVIEW

**Review by saramsey21 — APPROVED** [review]
database for user login

File: server/server.js [hunk 1]
```diff
@@ -1,9 +1,24 @@
 import express from "express";
+import mongoose from "mongoose";
 import bodyParser from "body-parser";
 import routesHandler from "./routes/handler.js"
 
 
 const app = express();
+
+const uri = 'mongodb+srv://lyfeonadmin:lfdPxA378lJdsQ9o@lyfeoncluster.7xvhbkd.mongodb.net/?retryWrites=true&w=majority'
```
Comment (davidcao1337) [inline server/server.js]: Uri should not be exposed

Comment (davidcao1337) [reply server/server.js]: Looking into using dotenv to hide

Comment (davidcao1337) [reply server/server.js]: Fixed

**Pull request merged**